model/k8s/metrics.yaml

```diff
@@ -469,3 +469,21 @@ groups:
     attributes:
       - ref: k8s.namespace.phase
         requirement_level: required
+
+  # k8s.container.* metrics
+  - id: metric.k8s.container.restarts
+    type: metric
+    metric_name: k8s.container.restarts
```

Comment:
Hmm, based on https://github.com/open-telemetry/semantic-conventions/blob/main/docs/general/naming.md#use-count-instead-of-pluralization-for-updowncounters this should be `k8s.container.restart.count` instead.

Reply:
However that's not the case for other updowncounters like https://github.com/open-telemetry/semantic-conventions/blob/main/model/k8s/metrics.yaml#L131-L142.

@open-telemetry/specs-semconv-approvers could you help here?  Should we always use count for updowncounters as the title from [Use count instead of pluralization for UpDownCounters](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/general/naming.md#use-count-instead-of-pluralization-for-updowncounters) implies?